examples/main/main.cpp

```diff
@@ -15,11 +16,15 @@
 #include <cstdio>
 #include <cstring>
 #include <ctime>
+#include <experimental/filesystem>
```

Comment:
My experience with `std::filesystem` is terrible - very poor compiler support.
Should avoid it

Reply:
Alright. It's annoying that there are compatibility issues when that is the whole purpose of the library but it should not be too hard to write the corresponding code myself.